OMCompiler/SimulationRuntime/c/util/simulation_options.h

```diff
@@ -189,6 +200,60 @@ extern const char *FLAG_DETAILED_DESC[FLAG_MAX+1];
 extern const flag_repeat_policy FLAG_REPEAT_POLICIES[FLAG_MAX];
 extern const int FLAG_TYPE[FLAG_MAX];
 
+enum GB_SINGLERATE_METHOD {
+  RK_UNKNOWN = 0,
```

Comment:
Add `GB_` (or maybe `GB_METHOD_`) before every enum to clarify that they relate to `GB_METHOD`.